core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -0,0 +1,468 @@
+                                return type === 'display' ? formatDuration(row.taskMetrics.executorRunTime) : row.taskMetrics.executorRunTime;
+                            }
+                        },
+                        {
+                            data : function (row, type) {
+                                return type === 'display' ? formatDuration(row.taskMetrics.jvmGcTime) : row.taskMetrics.jvmGcTime;
+                            }
+                        },
+                        {
+                            data : function (row, type) {
+                                return type === 'display' ? formatDuration(row.schedulerDelay) : row.schedulerDelay;
+                            }
+                        },
+                        {
+                            data : function (row, type) {
+                                return type === 'display' ? formatDuration(row.taskMetrics.executorDeserializeTime) : row.taskMetrics.executorDeserializeTime;
+                            }
+                        },
+                        {
+                            data : function (row, type) {
+                                return type === 'display' ? formatDuration(row.taskMetrics.shuffleReadMetrics.fetchWaitTime) : row.taskMetrics.shuffleReadMetrics.fetchWaitTime;
+                            }
+                        },
+                        {
+                            data : function (row, type) {
+                                return type === 'display' ? formatBytes(row.taskMetrics.shuffleReadMetrics.remoteBytesRead, type) : row.taskMetrics.shuffleReadMetrics.remoteBytesRead;
+                            }
+                        },
+                        {
+                            data : function (row, type) {
+                                return type === 'display' ? formatDuration(row.taskMetrics.resultSerializationTime) : row.taskMetrics.resultSerializationTime;
+                            }
+                        },
+                        {
+                            data : function (row, type) {
+                                return type === 'display' ? formatDuration(row.gettingResultTime) : row.gettingResultTime;
+                            }
+                        },
+                        {
+                            data : function (row, type) {
+                                return type === 'display' ? formatBytes(row.taskMetrics.peakExecutionMemory, type) : row.taskMetrics.peakExecutionMemory;
+                            }
+                        },
+                        {
+                            data : function (row, type) {
+                                if (accumulator_table.length > 0 && row.accumulatorUpdates.length > 0) {
+                                    return row.accumulatorUpdates[0].name + ' : ' + row.accumulatorUpdates[0].update;
+                                } else {
+                                    return 0;
```

Comment:
The should be `""` instead of `0` to match the previous page (similar to the error column below)

Reply:
Done